### PR TITLE
Provides default parameters for the RoadNetworkLoader.

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

Related to https://github.com/maliput/maliput/issues/531
Relies on https://github.com/maliput/maliput/pull/532

## Summary
Implements the new method for providing default parameters via RoadNetworkLoader.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: src/plugin/road_network.cc [hunk 1]
```diff
@@ -46,6 +47,9 @@ class RoadNetworkLoader : public maliput::plugin::RoadNetworkLoader {
       const std::map<std::string, std::string>& properties) const override {
     return malidrive::builder::RoadNetworkBuilder(properties)();
   }
+  std::map<std::string, std::string> GetDefaultParameters() const override {
+    return malidrive::builder::params::GetDefaultParameters();
```
Comment (agalbachicar) [inline src/plugin/road_network.cc:51]: Can't we do the same as in this function but here? It seems a trivial thing and we are adding a cc file to the library just for one line (without the boilerplate code).

**Review by agalbachicar — CHANGES_REQUESTED** [review]
PTAL

File: src/plugin/road_network.cc (hunk 1 above)
Comment (francocipollone) [reply src/plugin/road_network.cc]: Done.

**Review by agalbachicar — APPROVED** [review]
LGTM

**Comment by agalbachicar** [comment]
Check CI please.

**Comment by francocipollone** [comment]
@ros-pull-request-builder retest this please

**Comment by francocipollone** [comment]
@ros-pull-request-builder retest this please

**Pull request merged**